gradle/built-uber-dists/NOTICE-BINARY-DIST

```diff
@@ -0,0 +1,446 @@
+
+Dremio
+Copyright 2015-2025 Dremio Corporation
```

Comment:
Should the header be 1:1 with the root NOTICE?

nit: extra black line at the top.